builder/nodejs.yaml.in

```diff
@@ -0,0 +1,7 @@
+steps:
+  - name: 'gcr.io/$PROJECT/nodejs-gen-dockerfile:$TAG'
```

Comment:
I've updated the code.  Let me know if the update still looks incorrect, and we can discuss the correct way I should be implementing this file.